test/generated/elf.h

```diff
@@ -1 +1,75 @@
-// This is included by sail generated code but not actually needed
```

Comment:
These files aren't generated?

Reply:
They are derived from sail.

Reply:
I just had a stub one here - a later commit in this series copies the sail version over.

Reply:
Yes, but they're not generated, they're hand-written, and you're just vendoring them

Reply:
What would you suggest instead?

Reply:
I agree that generated isn't the best name - I just used that since the regen_* scripts create them.

Reply:
Have a test/{contrib,vendor,3rdparty,...}?

Reply:
Done